bitrise.yml

```diff
@@ -14,13 +14,15 @@ workflows:
   # ----------------------------------------------------------------
   # --- workflow to Step Test
   ci:
-    before_run:
-    - go-tests
     after_run:
     - test_ios
     - test_android_custom_options
     - test_macos
     steps:
+    - go-list:
```

Comment:
Please 
- update the yml format version.
- remove the `RELEASE_VERSION`
- remove the `create-release` & `share-this-step` wfs.